### ISSUE TITLE
Captum Occlusion produces exactly-zero attribution values on the example use case (unresolved)

### ISSUE THREAD

**Issue by antoinecomp** [antoinecomp/KAA#15]
## What happened

During the same validation run as #13/#14, once both were worked around, the pipeline ran to completion ("Actions end normally.") and produced real output files (inference JSON, explanation `.pkl`, plot PNGs, archived report) -- but the actual explanation data is degenerate:

```python
>>> data = pickle.load(open(".../200617-0852_3207236_-C20_OK.pkl", "rb"))
>>> data.shape
torch.Size([3, 224, 224])
>>> data.min(), data.max(), data.mean()
(0.0, 0.0, 0.0)
```

Every value in the attribution tensor is exactly `0.0` -- not small/noisy values consistent with a low-impact occlusion, but an exact-zero tensor. The generated plot PNG visually looks like the unmodified input image with no visible heatmap overlay, consistent with an all-zero attribution map.

## What is NOT yet known

This has not been root-caused. Candidate explanations, none confirmed:
- A bug in how `xData`/`target`/`baselines` are passed into `Captum`'s `Occlusion.attribute()` in `computeExplanationsImages` (`kaa/pluginsXAI/Captum/Captum_computeExplanations.py`, Occlusion branch, ~line 466-482).
- A genuine degenerate case for this specific fine-tuned ResNet50 + these two example images (e.g. saturated softmax output insensitive to any occlusion at this window/stride) -- would need checking the raw model output/target-class confidence for these two images.
- Something specific to the `sz`/`bs` workaround values used (`cowx:25 cowy:25 cosz:25 conp:1`, `bs` left at its default) -- not yet tried with different parameter values to see if zero-ness persists.

## Repro

- macOS arm64, Python 3.10, `torch==2.4.1`, `torchvision==0.19.1`, `captum==0.7.0`, `tensorflow-macos` (only installed to work around #13; unrelated to this issue).
- `UCRenaultWeldingResnet` + `Resnet` model, data mode `dm:1 dn:2` (the 2 bundled example images).
- Library: Captum, Method: Occlusion, params `wx=25 wy=25 sz=25 bs=None np=1`.
- Result: `.pkl` explanation tensors are all-zero for both example images.

## Next step

Before trusting any Captum/Occlusion result on real data (including tomorrow's company-model test), reproduce this in isolation: call `Occlusion(model).attribute(...)` directly on the loaded `RenaultWeldingResnet` model with the same inputs, outside of the KAA plumbing, to determine whether the zero-ness is in Captum's own computation (model/data issue) or in how KAA wires the call (a KAA bug in `computeExplanationsImages`).
